src/crimson/osd/object_context_loader.cc

```diff
@@ -8,6 +9,117 @@ namespace crimson::osd {
 
 using crimson::common::local_conf;
 
+
+ObjectContextLoader::load_and_lock_fut
+ObjectContextLoader::load_and_lock_head(Manager &manager, RWState::State lock_type)
+{
+  LOG_PREFIX(ObjectContextLoader::load_and_lock_head);
+  auto releaser = manager.get_releaser();
```

Comment:
Why do we need an additional releaser (which will be cancelled) on top of Manager's dtor?

Reply:
We actually don't, but it seemed easier to debug problems if load_and_lock_head doesn't leave partial locks if unsuccessful.